submissions/JuliaVorotchenko/Popup-task/main.css

```diff
@@ -0,0 +1,368 @@
+
+body{
+    font-family: 'Work Sans', sans-serif;
+
+}
+a {
+    text-decoration: none; 
+    color: #31353a;
+}
+
+li {
+    list-style-type: none;
+}
+
+.nav-wrapper {
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    display: flex;
+    justify-content: flex-end;
+}
+
+.navigation {
+    line-height: 30px;
+    display: flex;
+    align-items: center;
+    padding-top: 20px;
+    padding-right: 20px;
+}
+
+.nav-item {
+    padding-right: 25px;
+    transition: all 0.3s ease-in-out;
+}
+
+.text {
+    text-decoration: none;
+    font-family: 'Work Sans', sans-serif;
+    font-size: 20px;
+    color: #31353a;
+}
+
+.text:hover,
+.text:active,
+.text:focus { 
+    font-weight: 500;
+}
+
+.grid {
+    width: 28px;
+    height: 28px;
+}
+
+.bell {
+    width: 28px;
+    height: 28px;
+}
+
+.avatar {
+    padding-right: 10px;
+    display: block;
+    height: 45px;
+    width: 45px;
+    border-radius: 50%;
+    background: url(img/girl.svg);
+    background-size: cover;
+}
+
+.popup-wrapper {
+    display: none;
+    width: 320px;
+    height: auto;
+    position: absolute;
+    border: 1px solid #cccc;
+    background:#edf9f6;
+    box-shadow: 0 2px 10px #dae5e2;
+    z-index: 50;
+    right: 30px;
+}
+
+.app-container {
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+}
+
+.app-list {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 28px 15px 28px 18px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    justify-content: space-around;
+}
+
+.app-list-item {
+    width: 90px;
+    height: 90px;
+    padding: 2px;
+    display: flex;
+    justify-content: center;
+}
+
+.app-link {
+    width: 100%;
+    height: 100%;
+    border: 1px solid transparent;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: 'Work Sans', sans-serif;
+    color: #4a5b56;
+    text-decoration: none;
+    font-size: 400;
+}
+
+.app-list-item:first-child .app-icon {
+    background: url(img/gmail.svg);
+    background-size: cover;
+}
+
+.app-list-item:nth-child(2) .app-icon {
+    background: url(img/youtube.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(3) .app-icon {
+    background: url(img/hangouts.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(4) .app-icon {
+    background: url(img/photos.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(5) .app-icon {
+    background: url(img/calendar.svg);
+    background-size: contain;
+}
+
+.app-list-item:nth-child(6) .app-icon {
+    background: url(img/translate.svg);
+    background-size: contain;
+}
```

Comment:
In my opinion, it's better if you use the class name to read the code or future changes.